sw/device/tests/rv_core_ibex_icache_invalidate_test.c

```diff
@@ -0,0 +1,27 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/base/csr.h"
+#include "sw/device/lib/runtime/hart.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+
+// Initialize OTTF.
+OTTF_DEFINE_TEST_CONFIG();
+
+bool test_main(void) {
+  // Invalidate instruction cache 7 times.  (The number of iterations has to be
+  // kept in sync with the vseq).
+  for (unsigned i = 0; i < 7; i++) {
```

Comment:
small enhancement suggestion: you could use the testbench to backdoor overwrite this parameter via the backdoor symbol override DV infra, then you wouldn't have to worry about keep these in sync no?

Reply:
Thanks for the suggestion, I wasn't aware that this infra existed. I've changed it so that the vseq is the single source of truth for the expected number of icache invalidations (and that is now even randomized)